### PR TITLE
Overall installation advice

### PR DESCRIPTION
**Pull request by mpmdean**
Add sentence to help users decide how to install.

### CODE REVIEW

File: docs/source/user/installation.rst [hunk 1]
```diff
@@ -1,6 +1,13 @@
 ************
 Installation
 ************
+For those who just want to get started, using the :ref:`docker instructions <edrixsanddocker>` is relatively straightforward and is compatible with essentially all platforms. For linux users Anaconda_ is also easy to use. You can also compile the code from the source for Linux and OSX.
```
Comment (mrakitin) [inline docs/source/user/installation.rst]: Should not it be with an underscore?
```suggestion
For those who just want to get started, using the :ref:`docker instructions <_edrixsanddocker>` is relatively straightforward and is compatible with essentially all platforms. For linux users Anaconda_ is also easy to use. You can also compile the code from the source for Linux and OSX.
```

File: docs/source/user/installation.rst [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+
+
+Target Header goes here
+=======================
+
+Some text.
```
Comment (mrakitin) [inline docs/source/user/installation.rst]: Is this change needed?

File: docs/source/user/installation.rst (hunk 1 above)
Comment (mpmdean) [reply docs/source/user/installation.rst]: On my computer build passes without the underscore and fails with it. 

It might be that without the <> the trailing underscore is needed? Not sure.

File: docs/source/user/installation.rst (hunk 2 above)
Comment (mpmdean) [reply docs/source/user/installation.rst]: This was a straigh up screw up. I thought I'd removed this. I have removed it now

**Comment by mpmdean** [comment]
@mrakitin 
I think this is fixed now.

File: docs/source/user/installation.rst (hunk 1 above)
Comment (mrakitin) [reply docs/source/user/installation.rst]: It might not be needed for the section refs.

File: docs/source/user/installation.rst (hunk 2 above)
Comment (mrakitin) [reply docs/source/user/installation.rst]: Thanks!

**Comment by shenmidelin** [comment]
Thank you guys!
